packages/orchestration/src/types.d.ts

```diff
@@ -1,4 +1,10 @@
-import type { Amount, Brand, Payment, Purse } from '@agoric/ertp/exported.js';
+import type {
+  Amount,
+  Brand,
+  Payment,
+  Purse,
+  RemotableBrand,
```

Comment:
`RemotableBrand` comes from `@endo/eventual-send`, right? Importing it from ERTP is sort of misleading, suggesting it has something to do with the ERTP Brand type.

Reply:
`RemotableBrand` doesn't seem to be used. why add it?

Reply:
Artifact, removed. Thanks for surfacing